sdk/storage/ustorageclient/build_all/configs/ustorageclient_build_rules.cmake

```diff
@@ -0,0 +1,144 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#Use solution folders.
+set_property(GLOBAL PROPERTY USE_FOLDERS ON)
+
+# Build with -fPIC always
+set(CMAKE_POSITION_INDEPENDENT_CODE TRUE)
+
+option(run_valgrind "set run_valgrind to ON if tests are to be run under valgrind/helgrind/drd. Default is OFF" OFF)
+option(compileOption_C "passes a string to the command line of the C compiler" OFF)
```

Comment:
why is this needed? also the doc string is strange because this is a bool option so I don't understand which string gets passed.